src/components/rangeslider/attributes.js

```diff
@@ -30,6 +30,22 @@ module.exports = {
         role: 'style',
         description: 'Sets the border color of the range slider.'
     },
+    range: {
+        valType: 'info_array',
+        role: 'info',
+        items: [
+            {valType: 'number'},
+            {valType: 'number'}
+        ],
+        description: [
+            'Sets the initially selected range for the range slider.',
```

Comment:
Ahh I see - crossed wires on this one but that makes way more sense. I'll close this PR and open a new one once I've got it swapped about - shouldn't take too long.